services/ble/test_doubles/GattServerMock.hpp

```diff
@@ -11,7 +11,7 @@ namespace services
         : public GattServer
     {
     public:
-        MOCK_METHOD(void, AddService, (GattServerService& service));
+        MOCK_METHOD(void, AddService, (GattServerService & service));
```

Comment:
This one is weird. But automatic is better than manual.

Reply:
Indeed.